docs/sorting-algorithms/Quick-sort.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Quick Sort
+slug: /sorting-algorithms/Quick-sort
+---
+
+# Introduction
+
+Quick Sort is a sorting algorithm which follows Divide and Conquer. In this algorithm it picks an element as a pivot
+and arrange the array according to this pivot element.The condition to arrange the array according to pivot element
+is that all the elements which are smaller than or equal to pivot must lies on the left side of pivot.
+And all the elements which are greater than pivot must lie on the right side of the pivot. 
+This can be done in-place, requiring small additional amounts of memory to perform the sorting.  
+
+# Algorithm
+
+In Quick Sort, we follow the following algorithm:
+
+* first we have to pick a pivot element.
+
+* Then partition the array according to pivot element(smaller or equal elements than pivot must lies on left side and greater on right).
```

Comment:
`Partition the array according to the pivot element, where the smaller or equal elements compare to the pivot must lie on the left side and greater on the right side`